WSFrontend/org.openl.rules.ruleservice/src/org/openl/rules/ruleservice/loader/DeployStrategy.java

```diff
@@ -0,0 +1,33 @@
+package org.openl.rules.ruleservice.loader;
+
+import org.openl.util.BooleanUtils;
+import org.openl.util.StringUtils;
+
+public enum DeployStrategy {
+
+    /**
+     * Always deploys project to the production repository
+     */
+    ALWAYS,
+    /**
+     * Deploys is disabled
+     */
+    NEVER,
+    /**
+     * Deploys project to the production repository only if it is absent there
+     */
+    IF_ABSENT;
+
+    public static DeployStrategy fromString(String source) {
+        source = StringUtils.trimToNull(source);
+        if (source == null) {
+            return DeployStrategy.NEVER;
+        }
+        var booleanValue = BooleanUtils.toBooleanObject(source);
```

Comment:
Use Spring converter from String to Boolean there instead of OpenL utils

Reply:
done